### ISSUE TITLE
[Settings> Profiles]: Screen Reader is not announcing group label 'Profiles' with Profile tab item controls.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12005]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

Test Environment:
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Open Narrator using 'Win+Ctrl+ Enter' key. Turn OFF Narrator scan mode using 'Caps lock + Space' key.
4. Now navigate from Actions tab control to 'Defaults' control and so on, Then observe the issue.

**User Experience:**
Screen Reader users will be impacted here as they will not be able to structure complete information as label is not associated with controls. 

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[Screen Reader is not announcing group label 'Profiles' with Profile tab item controls..zip](https://github.com/microsoft/terminal/files/7750324/Screen.Reader.is.not.announcing.group.label.Profiles.with.Profile.tab.item.controls.zip)

### Expected Behavior

Screen Reader should announce group label 'Profiles' with tab item controls belongs to Profile group.
i.e. for 'Defults' tab item Screen Reader should announce 'Profiles Group Defaults 1 of 6 level 1'

### Actual Behavior

Screen Reader is not announcing group label 'Profiles' with tab item controls belongs to Profile group.
i.e. for 'Defaults' control Screen Reader is only announcing 'Defaults 1 of 6 level 1'
